config/locales/details.yml

```diff
@@ -128,8 +128,9 @@ en:
               penalty_level_2: £101 – £20,000
               penalty_level_3: £20,001 or more
             in_time:
-              'yes': 'Yes'
-              'no': 'No'
+              yes: No
```

Comment:
Can we add a comment here (or a key called _comment, since this is yaml) to make it clear why the sense of these answers is the other way around?